api/Dockerfile

```diff
@@ -12,8 +12,15 @@ FROM docker.io/realies/audiowaveform:1.10.1 AS awf
 
 # Identify dependencies of the `audiowaveform` binary and move them to `/deps`,
 # while retaining their folder structure
-RUN ldd /usr/local/bin/audiowaveform | tr -s '[:blank:]' '\n' | grep '^/' | \
-  xargs -I % sh -c 'mkdir -p $(dirname deps%); cp % deps%;'
+
+# Enable pipefail before `RUN` that contains pipes
+SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
+
+# hadolint ignore=SC2016
```

Comment:
What is this rule for? I couldn't find documentation for it. Also, should the comment on lines 13-14 moved here with it original code too?

Reply:
This particular rule is

```
SC2016 info: Expressions don't expand in single quotes, use double quotes for that.
```

Since we know that this script is perfectly fine, I decided to ignore that rule here.